quartz/plugins/transformers/ofm.ts

```diff
@@ -271,10 +282,10 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
           if (opts.parseArrows) {
             replacements.push([
               arrowRegex,
-              (_value: string, ..._capture: string[]) => {
+              (value: string, ..._capture: string[]) => {
                 return {
                   type: "html",
-                  value: `<span>&rarr;</span>`,
+                  value: `<span>${arrowMapping[value] || value}</span>`,
```

Comment:
i think let's check `arrowMapping[value]` exists outside of this and just don't replace if it's not a valid mapping

Reply:
updated @jackyzha0 ptal